clang/include/clang-c/Dependencies.h

```diff
@@ -288,18 +301,18 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v3(
  * \param [out] Out A non-NULL pointer to store the resulting dependencies. The
  *                  output must be freed by calling
  *                  \c clang_experimental_FileDependenciesList_dispose.
- * \param [out] error the error string to pass back to client (if any).
+ * \param [out] OutDiags The diagnostics emitted during scanning.
```

Comment:
Maybe mention that they need to be freed using `clang_disposeDiagnosticSet`.  I think it's particularly important since unlike the `error` out parameter you're setting these even when there is no error, so every client needs to be aware of them.

Reply:
Good catch. Mentioned that in the documentation.